### DISCUSSION TITLE
how to run test locally by Idea?

### DISCUSSION THREAD

**Discussion by laglangyue** [apache/pekko#883] [Q&A] [answered]
## env
openjdk version "17.0.9" 2023-10-17 LTS
OpenJDK Runtime Environment Corretto-17.0.9.8.1 (build 17.0.9+8-LTS)
OpenJDK 64-Bit Server VM Corretto-17.0.9.8.1 (build 17.0.9+8-LTS, mixed mode, sharing)
## run
![image](https://github.com/apache/incubator-pekko/assets/35491928/ecb855d3-4cef-49ec-b42b-33c73ed2ef3d)
![image](https://github.com/apache/incubator-pekko/assets/35491928/81f7a127-0a81-4a70-a11c-dc404ea63b68)

**Comment by laglangyue** [comment]
![image](https://github.com/apache/incubator-pekko/assets/35491928/b1742984-d203-42d7-892e-d23f80a9b44a)

**Comment by He-Pin** [comment, marked as answer]
Try use Java 8, in higher version of Java, you will need --add-exports and --add-opens
